api/graphql/resources/resource_types.py

```diff
@@ -18,3 +18,9 @@ class Meta:
             "buffer_time_before",
             "buffer_time_after",
         )
+
+        filter_fields = {
+            "name": ["exact", "icontains", "istartswith"],
+        }
+
```

Comment:
This change does not correspond to the commit title, and looks like it is adding a filter feature to the code base.